qiita_pet/static/js/networkVue.js

```diff
@@ -1144,8 +1144,10 @@ Vue.component('processing-graph', {
         '<td><small>Job status (circles):</small></td>' +
         '<td>' + circle_statuses.join('') + '</td>' +
         '<td rowspan="2" width="20px">&nbsp;</td>' +
-        '<td rowspan="2">' +
+        '<td rowspan="2">&nbsp;&nbsp;&nbsp;</td>' +
+        '<td rowspan="2" align="center">' +
             '<a class="btn btn-success form-control" id="add-default-workflow"><span class="glyphicon glyphicon-flash"></span> Add Default Workflow</a>' +
+             "<br/><a href='https://qiita.ucsd.edu/workflows/' target='_blank'>What's a Default Workflow?</a>" +
```

Comment:
I was thinking that if we are not vigilant enough with explaining the EMP assumptions of the Default Workflow we may get a lot of headaches later on when people not-knowingly process their non-EMP protocols. So anything we can to draw their attention to that important aspect I see as time-saving for admin later on.

One suggestion, what about replacing "What's a Default Workflow" to "Important note about Default Workflow"? <- That way they are more likely to read that description with the important note about the demux step. Or maybe adding a :warning: logo beside the button?